.gitignore

```diff
@@ -2,6 +2,7 @@ pjlib/include/pj/config_site.h
 lib/
 bin/
 output/
+Debug/

```

Comment:
what about "Release/" or any other build configs?
Or even better if Debug, Release, etc can be set to be under "output".

Reply:
It's actually used for "pjsip-apps\build\Debug" folder which contains some swig_java_pjsua2 output files. Need to change the project properties (swig_java_pjsua2) apparently